custom/enikshay/tasks.py

```diff
@@ -48,8 +48,11 @@ def enikshay_task():
     # `toggles.UATBC_ADHERENCE_TASK` enabled
     domains = toggles.UATBC_ADHERENCE_TASK.get_enabled_domains()
     for domain in domains:
-        updater = EpisodeUpdater(domain)
-        updater.run()
+        try:
```

Comment:
yeah, someone had added a bunch of domains without the right datasource into the feature flag, so this had just stopped running after the first failed domain.